extensions/ql-vscode/src/vscode-tests/no-workspace/distribution.test.ts

```diff
@@ -1,8 +1,18 @@
-import { expect } from "chai";
+import * as chai from "chai";
+import * as path from "path";
 import * as fetch from "node-fetch";
+import * as sinonChai from 'sinon-chai';
+import * as sinon from 'sinon';
+import * as pq from "proxyquire";
 import "mocha";
+
 import { Version } from "../../cli-version";
-import { GithubRelease, GithubReleaseAsset, ReleasesApiConsumer, versionCompare } from "../../distribution"
+import { GithubRelease, GithubReleaseAsset, ReleasesApiConsumer, versionCompare } from "../../distribution";
+
+const proxyquire = pq.noPreserveCache();
+chai.should();
```

Comment:
I'm told by chaijs.com/guide/styles this line can be replaced with `import 'chai/register-should'`;